### PR TITLE
add zkSync Era to Fees/Revenue dashboard

### PR DESCRIPTION
**Pull request by lgingerich**
FYI: had trouble with local testing

### CODE REVIEW

**Comment by llamatester** [comment]
The zksync-era adapter exports: 
        
 
 _______________________________________
    Fees for 20/8/2023
    _______________________________________
    
    Failed to get value on era: Error: Payment Required
    ERA 👇
    Timestamp attempted: 19/8/2023
    Error: Payment Required
        at retries (/home/runner/work/dimension-adapters/dimension-adapters/helpers/flipsidecrypto.ts:55:29)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async queryFlipside (/home/runner/work/dimension-adapters/dimension-adapters/helpers/flipsidecrypto.ts:11:10)
        at async fetch (/home/runner/work/dimension-adapters/dimension-adapters/fees/zksync-era/index.ts:23:28)
        at async /home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:32:52
        at async Promise.allSettled (index 0)
        at async /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:52:23
    
    
    
    
    Done in 4033.35s.

File: fees/zksync-era/index.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      fetch: async (timestamp: number) => {
+        const sql = postgres(process.env.INDEXA_DB!);
+        const now = new Date(timestamp * 1e3)
+        const dayAgo = new Date(now.getTime() - 1000 * 60 * 60 * 24)
+
+        try {
+          const fromTimestamp = timestamp - 60 * 60 * 24
+          const toTimestamp = timestamp
+
+          const startblock = (await getBlock(fromTimestamp, CHAIN.ERA, {}));
```
Comment (dtmkeng) [inline fees/zksync-era/index.ts]: use era block query on eth index is it correct ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: fees/zksync-era/index.ts (hunk 1 above)
Comment (lgingerich) [reply fees/zksync-era/index.ts]: Are you asking if the `getBlock()` usage is correct?

I copied it from the Arbitrum adapter and I don't see anything here or in the function definition that would prevent it from working.

Comment (dtmkeng) [reply fees/zksync-era/index.ts]: i mean you use block number of era query index eth
for arb it  use for query withdraw fees when team withdraw fees on fees wallet

Comment (lgingerich) [reply fees/zksync-era/index.ts]: Oh I see what you mean now — will fix. Thanks

**Comment by dtmkeng** [comment]
i think fees get from eth is not correct is there way to get on chain fees ?

**Comment by dtmkeng** [comment]
do you have result at yestday ? will verify result before live

**Comment by lgingerich** [comment]
This is aggregated by month and should be pretty accurate. https://dune.com/niftytable/rollup-economics

**Pull request merged**